fundrebalancer/fundrebalancer.go

```diff
@@ -630,16 +648,44 @@ func (r *FundRebalancer) estimateTotalGas(txns []SkipGoTxnWithMetadata) (uint64,
 	return totalGas, nil
 }
 
-func (r *FundRebalancer) isGasAcceptable(txns []SkipGoTxnWithMetadata, maxRebalancingGasThreshold uint64) (bool, uint64, error) {
-	// Check if total gas needed exceeds threshold to rebalance funds from this chain
-	totalRebalancingGas, err := r.estimateTotalGas(txns)
+// Compares total gas cost of fund rebalancing txns in USDC to configured threshold for a specified chain
+func (r *FundRebalancer) isGasAcceptable(ctx context.Context, txns []SkipGoTxnWithMetadata, chainID string) (bool, string, error) {
```

Comment:
just thinking out loud here, this introduces the potential that a fund rebalance never goes through if it does not meet the gas requirements. would it maybe make sense to add a timeout here like we have for settlements and a max max gas cost?  cc @dhfang